src/styles/partials/components/_site-header.scss

```diff
@@ -39,20 +39,34 @@
 }
 
 @media screen and (max-width: $header-breakpoint) {
```

Comment:
Recommend setting this to break at tablet size. At around 1024px (iPad size) the search bar/ `form` extends out of the frame of the header.

Reply:
> @martypowell Is there a class to toggle the search form in the space the beta text appears ie. open, active etc.? Trying to preview what it looks like when the magnifying glass icon is clicked. This looks good overall.

`is-visible`

Reply:
@theilman7 this is happening in the components project, but looks ok when applied to actual page.

Reply:
@martypowell As long as it doesn't extend out of frame when applied to the actual page, I'm fine with it. I know the components page has some weirdness when rendering some components within its containers.